### ISSUE TITLE
[1.1] When the server stops with a bag open, your inventory becomes the bag

### ISSUE THREAD

**Issue by smt923** [Chessnut/NutScript#533]
If I stop a server with a bag open, you lose most of your items and your inventory becomes just the bag, with no option to close it or anything, only way to fix it is it remake a character as far as I can tell

someone took a screenshot, they said there was no errors on their side (none on server either): 
http://images.akamai.steamusercontent.com/ugc/449581137799936684/DAC58E1B3E12ECD9D433F466E66E92CFEE500E0D/

**Comment by Chessnut** [comment]
Are there errors? This doesn't really make sense since the inventory
objects are completely separate.

**Comment by smt923** [comment]
I don't think there's any errors server side because the server literally stops straight away (I checked logs and couldn't find anything at least) and there was no errors when it came back up, clientside everyone says there was no error or anything, also searched the clientside_errors.txt and couldn't see anything related to bags, inventories, storage etc

**Comment by Chessnut** [comment]
This still doesn't make any sense since opening a bag is just client-side.

**Comment by smt923** [comment]
I'll test it a little more later, I'm not sure if anyone's tried fully restarting gmod but they've rejoined and it hasn't fixed it, same with switching chars etc, I seem to have a lot of errors that don't make sense

**Comment by smt923** [comment]
Annoyingly enough I can't actually recreate it, doesn't seem to happen to me when I copy what they've told me, unless it's something to do with that guys weird, pretty small res, but I suppose it's fine for me the few times I've tried it on my 1080p screen, though it's definitely happened too other people

**Comment by Chessnut** [comment]
Why didn't you just ask for a screenshot or video

**Comment by smt923** [comment]
I got a screenshot above, it's hard to take a video of it happening if they don't expect it to happen and can't go back to having a normal inventory to recreate it

**Comment by Chessnut** [comment]
Why not just have the person create a new character and record it? Tell him
when you're about to restart

**Comment by Chessnut** [comment]
Can't replicate this.

**Comment by NDKilla** [comment]
Not sure if it is related to this issue (I don't know if I restarted the server at the time).
@Chessnut when my server was in heavydevelopment I noticed a similar issue several times. I actually deleted a few of my legit characters because inventory became completely broken after adding an item to the server with a bag open and then restarting the server. Possibly not current since I haven't updated to the master branch in a while. I noticed similar problems though.

When I return home I guess I'll test it.
